.ci/e2e/publish_tests/func_tests_core/customer_churn_app/no_bundler_test.sh

```diff
@@ -11,6 +11,7 @@ FUNCTIONS_PYTHON_DOCKER_IMAGE=$4 "$3" azure functionapp publish "$2" --build-nat
 
 if [[ $? -eq 0 ]]
 then
+    sleep 5s
```

Comment:
Instead of 5s delay in the test, add delay 30sec right after func host start. add logging waiting for host to be ready

Reply:
@pragnagopa, these tests do not run `func host start`. They are testing publish actions.

Reply:
The delay is between publish and trying to check the trigger endpoint. It's not necessary, but in some cases I saw a second or two delay

Reply:
Sorry. I meant right after publish. Give 10sec-15sec before pinging

Reply:
never mind. i remember the flow now. each test is publish and ping?

Reply:
The reason I didn't is because I realized 1 second delay was more than enough. But adding 15 seconds to each tests would mean (15 seconds * 20 tests = 300 seconds) 5 extra minutes that is unnecessary. Specifically because if the host is busy starting up, `curl` would just wait to get a response because the request would not be completed. 

I would say we can leave it to 5sec for now. And if we see issues around this, it should be a quick update.

Reply:
yes. 5secs is ok.

Reply:
sorry didn't see your reply. Merging this.